Modules/README.md

```diff
@@ -9,5 +9,6 @@ sys-netcheat | netcheat sys-module by jakibaki
 edizon | EdiZon
 appstore | AppstoreNX
 Checkpoint | Save File Manager
-sdfilesupdater | The SDFilesUpdater homebrew
-hbmenu | The homebrew menu
\ No newline at end of file
+kosmossupdater | The Kosmos homebrew
```

Comment:
KosmosUpdater

Reply:
I meant the description as well here. It doesn't really describe it's purpose like the old one did. Maybe something like "A homebrew for directly updating Kosmos"

Reply:
OK i can do this